docker/build.sh

```diff
@@ -10,6 +10,7 @@ EMBED_USE_LLVM=${EMBED_USE_LLVM:-OFF}
 EMBED_BUILD_LLVM=${EMBED_BUILD_LLVM:-OFF}
 ALLOW_UNSAFE_PROBE=${ALLOW_UNSAFE_PROBE:-OFF}
 DEPS_ONLY=${DEPS_ONLY:-OFF}
+BUILD_TESTING=${BUILD_TESTING:-OFF}
```

Comment:
I think this needs to default to `ON`. Otherwise it disables all the tests in the CI.

```suggestion
BUILD_TESTING=${BUILD_TESTING:-ON}
```

Reply:
I see, let me enable back